pkg/web/apps/apps_http_handler.go

```diff
@@ -119,3 +120,48 @@ func (a *httpHandler) DisableAllAppVersionsByAppID(c echo.Context) error {
 	return c.JSON(http.StatusOK, "")
 
 }
+
+// InitClientApp stores device information and returns if the app version is disabled
+func (a *httpHandler) InitClientApp(c echo.Context) error {
```

Comment:
Note that is not required to create a new handler as you did. 
According to the @davidffrench defs I understand that just one is enough.